python/ray/train/BUILD

```diff
@@ -46,6 +46,66 @@ py_library(
     srcs = ["tests/conftest.py"]
 )
 
+############ Experiment tracking examples start ############
+
+# no credentials needed
+py_test(
+    name = "lightning_exp_tracking_mlflow",
+    size = "small",
+    main = "examples/experiment_tracking/lightning_exp_tracking_mlflow.py",
+    srcs = ["examples/experiment_tracking/lightning_exp_tracking_mlflow.py", "examples/experiment_tracking/lightning_exp_tracking_model_dl.py"],
+    tags = ["team:ml", "exclusive", "no_main", "new_storage"],
+    deps = [":train_lib"],
+)
+
+py_test(
+    name = "lightning_exp_tracking_tensorboard",
+    size = "small",
+    main = "examples/experiment_tracking/lightning_exp_tracking_tensorboard.py",
+    srcs = ["examples/experiment_tracking/lightning_exp_tracking_tensorboard.py", "examples/experiment_tracking/lightning_exp_tracking_model_dl.py"],
+    tags = ["team:ml", "exclusive", "no_main", "new_storage"],
+    deps = [":train_lib"],
+)
+
+py_test(
+    name = "torch_exp_tracking_mlflow",
+    size = "small",
+    main = "examples/experiment_tracking/torch_exp_tracking_mlflow.py",
+    srcs = ["examples/experiment_tracking/torch_exp_tracking_mlflow.py"],
+    tags = ["team:ml", "exclusive", "no_main", "new_storage"],
+    deps = [":train_lib"],
+)
+
+# credentials needed
+py_test(
+    name = "lightning_exp_tracking_wandb",
+    size = "small",
+    main = "examples/experiment_tracking/lightning_exp_tracking_wandb.py",
+    srcs = ["examples/experiment_tracking/lightning_exp_tracking_wandb.py", "examples/experiment_tracking/lightning_exp_tracking_model_dl.py"],
+    tags = ["team:ml", "exclusive", "no_main", "needs_credentials"],
+    deps = [":train_lib"],
+)
+
+py_test(
```

Comment:
I think we're missing the `lightning_exp_tracking_model_dl` one.

Can we keep these in alphabetical order?